dataland-frontend/src/components/forms/CreateLksgDataset.vue

```diff
@@ -119,6 +121,9 @@ import ProductionSitesFormField from "@/components/forms/parts/fields/Production
 import { objectDropNull, ObjectType } from "@/utils/UpdateObjectUtils";
 import { smoothScroll } from "@/utils/SmoothScroll";
 import { DocumentToUpload, uploadFiles } from "@/utils/FileUploadUtils";
+import MostImportantProductsFormField from "@/components/forms/parts/fields/MostImportantProductsFormField.vue";
+import ProductCategoriesFormField from "@/components/forms/parts/fields/ProductCategoriesFormField.vue";
```

Comment:
Rename to "ProcurementCategoriesFormField" ?

Reply:
Did that. Also changed the data dictionary. However I did not update PwC on the changes. I would wait for the merge of the branch before doing so.

Reply:
👍